libs/templates/src/lib/who-we-are/sections/mission-content/data.tsx

```diff
@@ -2,7 +2,7 @@ import { Text } from '@deriv/quill-design';
 
 const Content = () => {
   return (
-    <>
+    <div className="flex flex-col gap-gap-xl [&>p]:text-opacity-black-600">
```

Comment:
We can pass the `text-opacity-black-600` to the `className` props of the `Text` component.

Reply:
Done with added class into Text component